### DISCUSSION TITLE
Build a custom description for TFS item

### DISCUSSION THREAD

**Discussion by abaganuv** [DarqueWarrior/vsteam#461] [Q&A]
Hello
I want to build a custom description for TFS item that includes hyper link and bold / For an example

**The following fixed were added:**
fix1 - ""
fix2 - ""

I holds the title for each one of the rows in an object and want to run in a loop and fill the description 
There is a way to do it?

**Comment by SebastianSchuetze** [comment]
Hi @abaganuv are you talking about TFS (2017 / 2018) or about Azure DevOps. It is important, because it depends how the answer is (depending on the API version).

But in general you can do that probably with cmdlets for work items (e.g. Update-VSTeamWorkItem). The description field itself is just HTML based. So if you generate HTML from you code and save this in the description field, then you pretty much have what you want.

Although I am not sure why you would want to log fixes that you have done in a new work item.

**Reply by abaganuv** [reply]
Hi
Thank you for your answer, I am working with TFS 
I have a power shell script that read work items. I have an object that holds all the titles (If i will have an option to copy the link it will be better). What i need it a template as described above in bold and under all the titles (Row for each one)
But I don't know how to do it
My script build a new version with all new items and at the end I want to create a new task with all the new items that added
For -Title it is working great
I need a template for the -Description (From PowerShell)

**Reply by SebastianSchuetze** [reply]
Since it is open source and there is no guaranteed support. We would only be able to advice a bit but not write scripts for you. If you have no experience with PowerShell or VSTeam then I would suggest to read the documentation of the module and search for the right functions and then test.

You can also copy your script here. Because it could explain a bit better what you are doing currently.
